keel-api/src/test/resources/examples/cluster-example.yml

```diff
@@ -11,7 +11,7 @@ spec:
   imageProvider:
     deliveryArtifact:
       name: keeldemo
-      type: DEB
+      type: deb
```

Comment:
Look @erikmunson no more shouting, ARE YOU HAPPY!?

Reply:
THIS IS THE GREATEST DAY OF MY LIFE. I AM VERY HAPPY.

THANKS FOR ASKING.